experimental/FTheoryTools/docs/src/tate.md

```diff
@@ -27,12 +25,47 @@ In this 5-fold ambient space, a global Tate model is the hypersurface defined
 by the vanishing of the Tate polynomial
 ``P_T = x^3 - y^2 - x y z a_1 + x^2 z^2 a_2 - y z^3 a_3 + x z^4 a_4 + z^6 a_6``.
 
-Just as for Weierstrass models, our constructors construct for a given
-toric 3-fold one ambient space for the Tate model in question. As mentioned
-for Weierstrass models, there can exist other ambient spaces. Also, note that
-the ambient space constructed by our methods need not be smooth.
+Crucially, for non-trivial F-theory settings, the elliptic fibration in question must
+be singular. In fact, by construction, one usually engineers certain singularities.
+For this, vanishing orders of the sections ``a_i`` above need to specified. The
+following table -- often referred to as the Tate table and taken from [Wei10](@cite) -- 
+summarizes the singularities introduced by certain vanishing orders:
```

Comment:
```suggestion
following table---often referred to as the Tate table and taken from
[Wei10](@cite)---summarizes the singularities introduced by certain vanishing orders:
```
I don't know if the Oscar style guide has a preference on this... but I think asides within sentences should be set off with em dashes rather than en dashes (whether to surround them with spaces is yet another discussion to be had)

Reply:
I did not think deeply about it and just put -- there. Also, I am not aware if there is such a convention yet. In any case - I am happy with this change.